frontend/src/components/slider/index.jsx

```diff
@@ -0,0 +1,96 @@
+import { createSignal, createEffect, on } from "solid-js";
+import { styled } from "solid-styled-components";
+import arrow from "../../assets/arrow.svg";
+
+const Slider = (props) => {
+  const [imageIndex, setImageIndex] = createSignal(0);
+
+  // refs
+  let buttonNext, buttonPrev;
+
+  const imageList = () => props.imageList || [];
+  createEffect(on(imageList, () => setImageIndex(0)));
+
+  // disable arrows if no data after suppose handler
+  createEffect(() => {
+    const numImages = imageList().length;
+    buttonPrev.disabled = true;
+    buttonNext.disabled = true;
+    if (imageIndex() !== 0) {
+      buttonPrev.disabled = false;
+    }
+    if (imageIndex() < numImages - 1) {
+      buttonNext.disabled = false;
+    }
+  });
+
+  return (
+    <Wrapper>
+      <Image src={imageList()[imageIndex()]} />
```

Comment:
<img width="1171" alt="Screenshot 2023-03-22 at 00 25 21" src="https://user-images.githubusercontent.com/54143556/226755641-3eafaf76-0a80-4d3a-b84e-a49771adc374.png">
Please fix bug depicted on the image above